wikimedia/wikipedia-ios
# Move delete donation history to main settings VC

**Phabricator:** https://phabricator.wikimedia.org/T376219

### Notes
* Moves the logic that was under Settings > Account to the main settings view

### Test Steps
1. In settings, under "Privacy and terms," verify that you don't see the "Clear donation... " item 
2. Make a donation 
3. Go to Settings > account. You should not see the 'Clear donation...' button 
4. Go back to settings. You should now see the item under "Privacy and terms".
5. Click it and delete the history. 
6. The table view should reload and remove the 'Clear donation... " item
